.travis.yml

```diff
@@ -12,6 +12,9 @@ addons:
     - libreadline-dev:i386
     - gcc-multilib
     - g++-multilib
```

Comment:
You can also remove the :i386 and -multilib lines here, as no 32bit tests are performed anyway...